programming/hardware-support/mouse-support.rst

```diff
@@ -37,9 +37,10 @@ To get the position:
 
    .. code-block:: python
 
-      if base.mouseWatcherNode.hasMouse():
-        x = base.mouseWatcherNode.getMouseX()
-        y = base.mouseWatcherNode.getMouseY()
+      pointer = base.win.get_pointer(0)
+      if pointer.in_window:
+	      mouseX = pointer.get_x()
```

Comment:
bad indent, needs another space on this line

Let's encourage `pointer.x` and `pointer.y` instead

Variable names should be all-lower-caps, so use `mouse_x` and `mouse_y` (or just `x` and `y` as in the original)